docs/zkapps/how-to-write-a-zkapp.mdx

```diff
@@ -20,85 +20,124 @@ zkApps can now be deployed to Berkeley Testnet.
 
 # How to Write a zkApp
 
-A zkApp consists of a smart contract and a UI to interact with it. First, install the Mina zkApp CLI and write a smart contract.
+A zkApp consists of a smart contract and a UI to interact with it. 
 
-## Write a smart contract
+Write your smart contract using the [Mina zkApp CLI](https://www.npmjs.com/package/zkapp-cli/). 
 
-Write your smart contract using the [Mina zkApp CLI](https://www.npmjs.com/package/zkapp-cli/).
+First, install the Mina zkApp CLI.
 
-Mina zkApp CLI makes it easy to follow recommended best practices by providing project scaffolding including dependencies such as SnarkyJS, a test framework (<a href="https://jestjs.io/">Jest</a>), code auto-formatting (<a href="https://prettier.io/
-">Prettier</a>), linting (<a href="https://eslint.org/">ES Lint</a>), & more.
+The Mina zkApp CLI makes it easy to follow recommended best practices by providing project scaffolding including dependencies such as SnarkyJS, a test framework [Jest](https://jestjs.io/), code auto-formatting [Prettier](https://prettier.io/
+), linting [ES Lint](https://eslint.org/), and more.
 
-### Install Mina zkApp CLI
+## Install Mina zkApp CLI
 
 ```sh
 npm install -g zkapp-cli
 ```
 
-#### Dependencies:
+Dependencies:
 
 - NodeJS 16+ (or 14 using `node --experimental-wasm-threads`)
 - NPM 6+
 - Git 2+
 
-:::tip
+If you have a later version installed, install the required version using the package manager for your system: 
 
-If you have an older version installed, install a newer version using the package manager for your system: <a href="https://brew.sh/">Homebrew</a> (Mac), <a href="https://chocolatey.org/">Chocolatey</a> (Windows), or apt/yum/etc (Linux). On Linux, you may need to install a recent NodeJS version via NodeSource (<a href="https://github.com/nodesource/distributions#debinstall">deb</a> or <a href="https://github.com/nodesource/distributions#rpminstall">rpm</a>), as recommended by the NodeJS Project.
+- MacOs [Homebrew](https://brew.sh/)
 
-:::
+- Windows [Chocolatey](https://chocolatey.org/)
+
+- Linux (apt, yum, and others)
+
+    As recommended by the Node.js project, you might need to install a recent Node.js version using NodeSource binary distributions: [Debian](https://github.com/nodesource/distributions#debinstall), [rpm](https://github.com/nodesource/distributions#rpminstall).
+
+## Start a project
+
+Now that you have the Mina zkApp CLI installed, you can start with an example or start your own project.
+
+### Option A: Start with an example (recommended)
+
+Examples are based on the standard project structure and provide additional files in the `/src` directory.
+
+1. Create the sudoku example project: `zk example sudoku`
```

Comment:
If a user runs  the `zk example` command,  an interactive prompt to choose an example is displayed to the user.
```
? Choose an example …
❯ sudoku
  tictactoe
```
@barriebyron do you think we should highlight the interactive commands `zk example` or the non-interactive commands `zk example sudoku`, or mention both throughout the docs?

Reply:
Good suggestion, I like helping the user learn about other examples, let's do the interactive step.